data/os/Fedora.yaml

```diff
@@ -1,5 +1,4 @@
 ---
 koji::builder::imaging_packages:
     - lorax
-    - pycdio
-    - python-kickstart
```

Comment:
Are you sure about this?  I merged this and am now trying to create a Fedora 30 based Builder.  Using python2 (as packaged) a test spin-livemedia task failed with `PreBuildError: Live Media functions not available` and looking at kojid, I see:
~~~
# imports for LiveCD, LiveMedia, and Appliance handler
try:
    import pykickstart.parser as ksparser
    import pykickstart.handlers.control as kscontrol
    import pykickstart.errors as kserrors
    import iso9660 # from pycdio
    image_enabled = True
except ImportError:  # pragma: no cover
    image_enabled = False
~~~

To my eye, it looks like both `pycdio` and `python-kickstart` are necessary for these imports to pass so that the boolean is set such that this error is avoided.  Looking at an older F28-based Builder I see essentially the same.

Reply:
Looks like they are, but you will need to adapt them for Python 2 or Python 3 depending on distro and release. In Mageia 7, it's all Python 3. Same would hopefully go for Fedora 30+.

Reply:
Ok, I will re-add them for Fedora here.  Mageia gets its own list of packages via `data/os/Mageia.yaml` (that you added) so that should remain unaffected.  (All Py3 sounds dreamy.  :smile: )

Reply:
Partially reverted with commit 9fc2bd0931a91.